### ISSUE TITLE
No source code

### ISSUE THREAD

**Issue by ghost** [OpenResearchInstitute/ribbit#2]
Hi,
you announce open source on your website but the code is nowhere to be found.

open source is about release often, release early, so the community can have a look at the progress.

Can you please share the source code of the app? A github project is for code right? not just a catchy readme... and a GPL licence. Is the readme under GPL? come on... Do you have secrets to work out before being able to show code?

Also ham radio is about learning and education, not just consuption. Specifically when you advertise a tool for disaster recovery that everyone should be able to use.

thanks for every ham that is waiting for this project.

**Comment by MisterPhocks** [comment]
Agreed, I was hoping I could take the code and translate it into Swift to use on my Apple devices but I can't find a single line of code anywhere.

**Comment by Abraxas3d** [comment]
Greetings all.

As has been announced, here's the links. Very sorry you couldn't find them
from the various announcements or a google search. This repo is for a
general program description and upcoming repeater-level work.

Source code for Rattlegram application on Android can be found here:

https://github.com/aicodix/rattlegram

core Rattlegram modem is here:

https://github.com/aicodix/modem

If you want the Polar Code library itself, then this link will be of
interest:

https://github.com/xdsopl/polar

Ribbit is the program. Rattlegram is the application.

Learn more about the Ribbit project at https://ribbitradio.org

Also, please consider joining the Ribbit mailing lists? That way you can
get informed directly. Anyone with more than passing interest in the
program and applications will be on at least one of these lists.

https://www.openresearch.institute/mailing-lists/

-Michelle Thompson
